### DISCUSSION TITLE
posts/how-i-use-org-roam-to-takes-notes-for-cs/

### DISCUSSION THREAD

**Discussion by giscus** [michaelneuper/website#57] [General]
# posts/how-i-use-org-roam-to-takes-notes-for-cs/

The aim of this post is to show how I use Doom Emacs
and org-roam to take notes for my Computer Science degree.

https://michaelneuper.com/posts/how-i-use-org-roam-to-takes-notes-for-cs/

**Comment by dmitrym0** [comment]
> At the end of the day, once I’m home and have some quiet time, I sit down to make my own notes. By then, I’ve already formed a basic understanding of the topic from the lecture. I start by writing down what I remember from class, which helps reinforce the key points. If I realise I’ve forgotten something or need clarification, I consult the textbook, lecture notes, or other resources to fill in the gaps.


This is so great. I wish I had the discipline to do that back in school.

**Comment by mohaids** [comment]
What font do you use?

**Reply by mohaids** [reply]
Never mind - I think I got it: Latin Modern Roman

**Comment by agzam** [comment]
I'm so fucking jealous, to be honest. I wish someone told me about Emacs back when I was a student. It took me over a decade of working professionally as a programmer to realize the crucial importance of good note taking. And I tried all sort of apps before finally settling with Org and later Org-Roam. I can only imagine the great things you'd able to learn and retain in your brain thanks to the great Emacs ecosystem. You have made the best choice, with far going implications. I promise you, not once, not even subtly, have I ever regretted having invested in learning Emacs. It's the best gift anyone can present to themself. You will forever feel indebted to your younger self for this gift - I guarantee that. Set a reoccurring calendar notification for every ten years, check this assumption, and you will see it.

**Comment by santiago-aghemo** [comment]
absolutely amazing post. saving this for the future. starting uni next year!
